deploy/eck-operator/templates/configmap.yaml

```diff
@@ -11,7 +11,9 @@ data:
     log-verbosity: {{ int .Values.config.logVerbosity }}
     metrics-port: {{ int .Values.config.metricsPort }}
     container-registry: {{ .Values.config.containerRegistry }}
-    container-suffix: {{ .Values.config.containerSuffix }}
+    {{- with .Values.config.containerSuffix }}
+    container-suffix: {{ . }}
+    {{- end }}
     max-concurrent-reconciles: {{ int .Values.config.maxConcurrentReconciles }}
     {{- if .Values.config.passwordHashCacheSize }}
```

Comment:
Do we want to change all of these `if` blocks to `with` blocks for consistency?

https://helm.sh/docs/chart_template_guide/control_structures/
`the block after with only executes if the value of PIPELINE is not empty.`

Reply:
I find the `with` syntax very nice for readability and figured it was worth starting to use it at the cost of a little break in consistency.

It would be nice to move all the `if` to `with` but this is a low hanging fruit: https://github.com/elastic/cloud-on-k8s/issues/6716.